README.md

```diff
@@ -36,3 +34,14 @@ curl --request POST \
 --header 'Content-Type: application/xml' \
 --data '@src/main/resources/examples/DO1-IM-request.xml'
 ```
+
```

Comment:
Good with changelog. Consider moving that to a CHANGLOG.md instead. Minor, not importan but if it gets long simplifies reading etc